src/App.js

```diff
@@ -17,6 +19,7 @@ function App() {
         >
           Learn React
         </a>
+        <Button variant="primary">Primary</Button>
```

Comment:
delete this no need to add in app.js will give us errors

Reply:
Done